core/chains/evm/logpoller/log_poller_internal_test.go

```diff
@@ -262,7 +262,7 @@ func TestLogPoller_Replay(t *testing.T) {
 
 	// process 1 log in block 3
 	lp.PollAndSaveLogs(testutils.Context(t), 4)
-	latest, err := lp.LatestBlock()
+	latest, err := lp.LatestBlock(lp.ctx)
```

Comment:
It is odd to pass an unexported field from lp back in through a method on lp. Let's store the `testutils.Context(t)` from the preceding line and use that here:
```suggestion
	latest, err := lp.LatestBlock(ctx)
```